includes/Admin/Admin.php

```diff
@@ -128,10 +128,15 @@ public function admin_body_class( $class ) {
 	 * @since 1.0.0
 	 *
 	 * @param string|mixed $content Default post content.
-	 * @param WP_Post      $post    Post object.
+	 * @param WP_Post|null $post    Post object.
 	 * @return string|mixed Pre-filled post content if applicable, or the default content otherwise.
 	 */
-	public function prefill_post_content( $content, WP_Post $post ) {
+	public function prefill_post_content( $content, ?WP_Post $post ) {
+		// When $post is NULL, set -1 to suppress phpstan error: Parameter #1 $post of function use_block_editor_for_post expects int|WP_Post, WP_Post|null given.
+		if ( \is_null( $post ) ) {
+			$post = -1;
+		}
```

Comment:
Hmm let‘s do `if ( ! $post ) { return $content; }` to be on the safe side. Because if we don't have a proper post object, then there's something wonky going on and we should just abort.

```suggestion
		if ( ! $post ) {
			return $content;
		}
```

Reply:
Seems like a good plan!